packages/shared/src/s3/commands.ts

```diff
@@ -35,6 +42,37 @@ import {
   GetBucketPolicy,
 } from './types';
 
+const s3ClientSigner = (
+  region: string,
+  credentials: AwsCredentialIdentity
+): RequestSigner => {
+  const signer = new SignatureV4({
+    credentials: credentials,
+    region: region,
+    service: 's3',
+    sha256: Sha256,
+  });
+
+  return {
+    sign: (request: HttpRequest) => {
+      const originalPath = request.path;
+      request.path = originalPath.replace(
+        '/api/proxy/plugin/odf-console/s3-proxy',
+        ''
+      );
+      request.headers['host'] = request.headers['host'].includes('localhost')
+        ? 'localhost:6001'
+        : (request.headers['host'] =
+            's3.openshift-storage.svc.cluster.local:443');
```

Comment:
aren't both syntax same ?
```suggestion
        : s3.openshift-storage.svc.cluster.local:443';
```

Reply:
Yes, that was accidental (due to previous experiments).